mercury/tests/selenium_tests/test_create_event.py

```diff
@@ -0,0 +1,49 @@
+from mercury.tests.selenium_tests.selenium_base import SeleniumTestBase
+from django.urls import reverse
+
+# from selenium import webdriver
+
+from selenium.webdriver import Firefox
+from selenium.webdriver.firefox.options import Options
+
+
+class CreateEventTest(SeleniumTestBase):
+    def setUp(self):
+        self.set_test_code()
+        # self.wd = webdriver.Firefox()
+        options = Options()
+        options.add_argument("-headless")
+        self.wd = Firefox(firefox_options=options)
+        self.wd.implicitly_wait(10)
+
+    def tearDown(self):
+        self.wd.quit()
+
+    def test_create_event(self):
+        # login with testcode
+        self.login_test_code()
+
+        # create a new event
+        self.open(reverse("mercury:event"))
+        self.wd.find_element_by_id("id_event_name").send_keys("event1")
+        self.wd.find_element_by_id("id_event_location").send_keys("New York")
+        self.wd.find_element_by_id("post-event-date").send_keys("2020-03-03")
+        self.wd.find_element_by_id("post-event-comments").send_keys("event1 comment")
```

Comment:
Hi I think this part is not corresponding to the current database schema

Reply:
@Ricardo-1894 yes indeed - do you know when those changes to the models will get into the master?

Reply:
@daisycrego You can find these changes in the `[backend-team](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/tree/backend-team)` branch now, we will probably merge changes into master later.

Reply:
@Ricardo-1894 awesome! we will close this PR for now